uber/custom_tags.py

```diff
@@ -16,7 +16,7 @@ def timestamp(dt):
 
 @register.filter
 def jsonize(x):
-    return SafeString(json.dumps(x, cls=serializer))
+    return SafeString(html.escape(json.dumps(x, cls=serializer), quote=False))
```

Comment:
I'm still a little dubious about this but it's better than being what it is now so +1

I think where possible we should be escaping the individual things that are getting serialized which would mean we'd need to write our own custom serializer.  Which is a whole thing.